internal/p2p/peermanager.go

```diff
@@ -38,11 +38,20 @@ const (
 	PeerStatusBad  PeerStatus = "bad"  // peer observed as bad
 )
 
+type peerConnectionDirection int
+
+const (
+	peerConnectionInvalid peerConnectionDirection = iota
+	peerConnectionIncoming
+	peerConnectionOutgoing
```

Comment:
Not necessary, but note also that you could say:
```suggestion
	peerConnectionIncoming peerConnectionDirection = iota + 1
	peerConnectionOutgoing
```
if you don't otherwise need the "invalid" label for anything. (The whole iota expression percolates down)

Reply:
oh nice.

Reply:
Fixed it in both branches.